curiouslearning/FeedTheMonsterJS
# feat: Adjusted the hitbox coordinates of where the stones should be dropped (FM-266).

# Changes
- Moved the handling if the stones dropped on the correct coordinates from gameplay-scene.ts to monster.ts.

# How to test
- Run locally, then start the game.
- Select any level and during the gameplay, dropping of stones should be withing or very close to the monster's face.
- Repeat this process on different screen sizes.

Ref: [FM-266](https://curiouslearning.atlassian.net/browse/FM-266)


[FM-266]: https://curiouslearning.atlassian.net/browse/FM-266?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
